examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +57,59 @@ describe('file-search', function () {
 
         });
 
+        describe('#buildOpenerOptions', () => {
+            it('should return the correct editor opener options', () => {
+                const rangeTest = {
+                    start: { line: 3, character: 3 },
+                    end: { line: 5, character: 5 }
+                };
+                quickFileOpenService['filterAndRange'].range = rangeTest;
+                expect(quickFileOpenService['buildOpenerOptions']().selection).eq(rangeTest);
+            });
```

Comment:
I don't see how this test will pass?
If no search is performed shouldn't the `filterAndRange` be the default:

```ts
protected filterAndRangeDefault = { filter: '', range: undefined };
```

Reply:
The test passes because I'm modifying the filterAndRange.range with this line :
`quickFileOpenService['filterAndRange'].range = rangeTest`
so it is not the default filterAndRange anymore.